pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -935,10 +965,11 @@ def get_instancename(context, instancename, options, ns_names=None):
                 # unprintable characters, and no quotes that might interfere
                 # with the shell.
                 kb_value = '"{}"'.format(mof_escaped(value))
+
             kb_strs.append("{}={}".format(key, kb_value))
 
-        # We perform an extra verification that instancename was a class path,
-        # in order to get a more understandable error message if it is not,
+        # We perform an extra verification that instancename was an instance
+        # path in order to get a more understandable error message if it is not,
```

Comment:
The following test is whether it is a class path, so the original comment was correct.

Reply:
DONE